### ISSUE TITLE
Translation fr-fr

### ISSUE THREAD

**Issue by cinqmilleans** [jurialmunkey/skin.eminence.2#117]
I see that several translations have been waiting for integration for a long time. I can translate Eminence into French, but I would like to know if you are interested before starting the translation.

**Comment by GulGreg** [comment]
Strange, there is no resource.language.fr_FR in code but my eminence is in French...

I can give an hand if needed to translate the skin.

**Comment by jurialmunkey** [comment]
> I see that several translations have been waiting for integration for a long time. I can translate Eminence into French, but I would like to know if you are interested before starting the translation.

Yeah I'll happily merge translations. I missed the hungarian one so going to merge that now. The other CJK one is a bit more complicated because it changes the skin design, which is why I didn't merge that one.

**Comment by maghiel** [comment]
@cinqmilleans @Dukat9063 do you still need french to be included? If so I'd be more than happy to merge it into my fork as this project seems unmaintained.

**Comment by Maaaash32** [comment]
Hello, I'm french and I use Eminence since 8 years. I can navigate in english but if the translation is already done I think it'd be usefull to merge it.

**Comment by maghiel** [comment]
@Maaaash32 if you can point me to a finished translation I'll merge it!
